pkgs/top-level/python-packages.nix

```diff
@@ -1979,17 +1983,17 @@ in {
   distributed = buildPythonPackage rec {
```

Comment:
This expression needs to be moved to `pkgs/development/python-modules/distributed/default.nix`.